pyqtgraph/examples/jupyter_console_example.py

```diff
@@ -23,7 +23,7 @@
 
 try:
     from qtconsole import inprocess
-except:
+except ImportError or NameError:
```

Comment:
I don't think the `or` operator does what you want it to.
```python
class A(Exception): pass
class B(Exception): pass

try:
    raise B()
except A or B:
    print('caught')
except:
    print('oops')
```

Reply:
Ahh, correct, what is needed here is `except (ImportError, NameError):

https://docs.python.org/3/tutorial/errors.html#handling-exceptions

Reply:
done!